gloom/srv/shopsdk.py

```diff
@@ -0,0 +1,143 @@
+import pickle
+import binascii
+import json
+import os
+import httpimport
+from sendgrid import (
+  SendGridAPIClient
+)
+from sendgrid.helpers.mail import (
+Mail, 
+Attachment, 
+FileContent, 
+FileName, 
+FileType, 
+Disposition
+)
+class sdk():
+  def send(thetoemail, filetosend, currentnoofpoints, pointsneeded, number, mime):
+    url = "https://raw.githubusercontent.com/6100m/room-server-1/hotfix/rc24/utils/by/larsen/rc24/utilsbylarsen.py"
+    with httpimport.remote_repo(["utilsbylarsen"], url):
+      import utilsbylarsen
+    with open(str(os.getcwd()) + "/" + "./config.json", "rb") as f:
+      config = json.load(f)
+    if conf["production"] and conf["send_logs"]:
+      utilsbylarsen.setup_log(config["sentry_url"], False)
+    if number == 1:
+      contenttype = "application/" + mime
+    elif number == 2:
+      contenttype = "video/" + mime
+    elif number == 3:
+      contenttype = "audio/" + mime
+    elif number == 4:
+      contenttype = "image/" + mime
+    elif number == 5:
+      contenttype = "text/" + mime
+    elif number == 6:
+      contenttype = "font/" + mime
+    else:
+       utilsbylarsen.log("THREW EXCEPTION IN ENGINE SCRIPT BECAUSE INTEGER IS: " % number, "CRITICAL")
+       raise ExceptionError
```

Comment:
We should have whatever file specify its own Content-Type individually instead of it potentially being deterministic here.

Reply:
I actually did that for ease of typing, those are the only 6 ones I think. I'll probably verify this later.